Source/Lib/Codec/EbEncHandle.c

```diff
@@ -2505,20 +2505,19 @@ void SetParamBasedOnInput(
     sequenceControlSetPtr->topPadding   = MAX_LCU_SIZE + 4;
     sequenceControlSetPtr->rightPadding = MAX_LCU_SIZE + 4;
     sequenceControlSetPtr->botPadding   = MAX_LCU_SIZE + 4;
-
     sequenceControlSetPtr->lumaWidth    = sequenceControlSetPtr->maxInputLumaWidth;
     sequenceControlSetPtr->lumaHeight   = sequenceControlSetPtr->maxInputLumaHeight;
 
     chromaFormat = sequenceControlSetPtr->chromaFormatIdc;
     subWidthCMinus1 = (chromaFormat == EB_YUV444 ? 1 : 2) - 1;
     subHeightCMinus1 = (chromaFormat >= EB_YUV422 ? 1 : 2) - 1;
 
-    sequenceControlSetPtr->chromaWidth  = sequenceControlSetPtr->maxInputLumaWidth >> subWidthCMinus1;
+    sequenceControlSetPtr->chromaWidth = sequenceControlSetPtr->maxInputLumaWidth >> subWidthCMinus1;
     sequenceControlSetPtr->chromaHeight = sequenceControlSetPtr->maxInputLumaHeight >> subHeightCMinus1;
 
-    sequenceControlSetPtr->padRight             = sequenceControlSetPtr->maxInputPadRight;
-    sequenceControlSetPtr->croppingRightOffset  = sequenceControlSetPtr->padRight;
-    sequenceControlSetPtr->padBottom            = sequenceControlSetPtr->maxInputPadBottom;
+    sequenceControlSetPtr->padRight = sequenceControlSetPtr->maxInputPadRight;
+    sequenceControlSetPtr->croppingRightOffset = sequenceControlSetPtr->padRight;
+    sequenceControlSetPtr->padBottom = sequenceControlSetPtr->maxInputPadBottom;
     sequenceControlSetPtr->croppingBottomOffset = sequenceControlSetPtr->padBottom;
```

Comment:
I'd recommend don't change the format of other code to avoid any possible rebasing of other PR.

Reply:
OK